src/main/java/com/awesomeorg/airlineservice/controller/BaggageController.java

```diff
@@ -12,4 +18,12 @@
 public class BaggageController {
```

Comment:
What about endpoints for removing baggage, listing baggage, and getting baggage for the reservation?